packages/generate/src/syntax/insert.ts

```diff
@@ -252,10 +252,12 @@ export function $normaliseInsertShape(
     const isMulti =
       pointer.cardinality === Cardinality.AtLeastOne ||
       pointer.cardinality === Cardinality.Many;
-    if (pointer.__kind__ === "property") {
-      if (pointer.target.__name__ === "std::json") {
-      }
-    }
+
+    // Todo delete or fix this
+    // if (pointer.__kind__ === "property") {
+    //   if (pointer.target.__name__ === "std::json") {
+    //   }
+    // }
```

Comment:
ha! Yeah, just delete it, it obviously isn't needed and if it's a bug, having this no-op block of code won't really help us fix the bug.